evap/staff/templates/staff_course_comments_quick.html

```diff
@@ -0,0 +1,285 @@
+{% extends 'staff_course_base.html' %}
+
+{% load evaluation_filters %}
+
+{% block breadcrumb %}
+    {{ block.super }}
+    <li class="breadcrumb-item">{% trans 'Comments' %}</li>
+{% endblock %}
+
+{% block content %}
+    {{ block.super }}
+
+    <div class="d-flex">
+        <h3 class="mr-auto">{{ course.name }} ({{ course.semester.name }})</h3>
```

Comment:
can you please move this part and the one from `staff_course_comments.html` to a common base template?